### PR TITLE
Default assets

### PR DESCRIPTION
**Pull request by Thykof**
On buildnet, you can QA with this token: AS1dTb8iM9ED2adV6a26BMjg3sYQywiyLn33nJTRZ2fiY7FtRGSA

(total supply is 1, with 9 decimals, I can send you some if needed)

The tokens are sorted alphabetically to have a constant order each time, we will sort is by dollar value when it will be available

### CODE REVIEW

**Comment by Thykof** [comment]
@pivilartisant on top of this Pull Request you can work on the dollar value display in the frontend, maybe on another branch

File: pkg/testUtils/testUtils.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	"github.com/massalabs/station-massa-wallet/pkg/wallet"
+	"github.com/massalabs/station/pkg/logger"
+)
+
+const (
+	LogFileNameTest = "station-massa-wallet-test.log"
+)
+
+func LoggerTest() error {
+	logPath, err := wallet.Path()
```
Comment (thomas-senechal) [inline pkg/testUtils/testUtils.go:15]: I would change that to write the logs in the repository directly
Would be way easier to check the logs of tests

**Review by thomas-senechal — COMMENTED** [review]
Only one small comment

File: pkg/testUtils/testUtils.go (hunk 1 above)
Comment (Thykof) [reply pkg/testUtils/testUtils.go:15]: ho yes!

**Review by thomas-senechal — APPROVED** [review]

**Review by pivilartisant — APPROVED** [review]

**Review by gregLibert — COMMENTED** [review]
Several observations:
- The non-compliance with the principles of **Separation of Concerns** and **Open/Closed**, or **Protected Variation**, suggests that this PR introduces legacy, unclean code, and potential regressions.
- It is not explained how the publication of new assets will be managed without causing breaking changes.

That said, refactoring the code will likely be too burdensome given the deadlines.

**Comment by Thykof** [comment]
> Several observations:
> 
>     * The non-compliance with the principles of **Separation of Concerns** and **Open/Closed**, or **Protected Variation**, suggests that this PR introduces legacy, unclean code, and potential regressions.
> 
>     * It is not explained how the publication of new assets will be managed without causing breaking changes.
> 
> 
> That said, refactoring the code will likely be too burdensome given the deadlines.

Did you have a look at the following PR https://github.com/massalabs/station-massa-wallet/pull/933?

**Pull request merged**